engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/module/scope/definition/Method.scala

```diff
@@ -49,13 +49,34 @@ object Method {
     */
   sealed case class Explicit(
     override val methodReference: Name.MethodReference,
-    override val body: Expression,
+    val bodySeq: Seq[Expression],
```

Comment:
Just leaving it like that is error-prone. Someone will forget that access to the body triggers the serialization logic and it will hurt the performance. It should be reflected in the type signature

Reply:
I wanted to write a test to verify that the _deserialization is lazy_. I may do it in a subsequent PR.